### PR TITLE
Add quickstart method

### LINKED ISSUE #245: Add a "quick start guide" episode
**Short description of the material**

Hi @ESMValGroup/userengagementteam,

I would like to propose adding a "quick start guide" episode.

The structure of this episode would be:

* Activate environment
* Configure ESMValTool
* Run example recipe

Having a "quick start guide" like this would lower the entry barrier for new users and would enable a user to quickly get started using ESMValTool, especially in those cases where they are working somewhere that already has ESMValTool installed and has data available (e.g. JASMIN, DKRZ, Met Office). The above sections would refer to the ESMValTool documentation for further information / incase the user is working in a different location.

We already have a draft "quick start guide" (developed by @mo-gregmunday) that we have been using at the Met Office, and @mo-tgeddes would create the PR, provided there are no objections :)

**Branch and pull request**
Once you've started working, add the branch (and pull request) link.

### PR DESCRIPTION
**Pull request by mo-tgeddes**
Adds a quick-start method for setting up the ESMValTool. Currently at the very beginning of the tutorial.
 
# Pull Request checklist

We appreciate your time and effort to improve the tutorial. Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution.

* * *

## Before you start

- [x] Read [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md).
- [x] Create an [issue](https://github.com/ESMValGroup/tutorial/issues) to discuss your idea. This allows your contributions to be incorporated into the tutorial.

## Tasks

- [x] Give this pull request a descriptive title.
- [x] If you are contributing to existing lesson materials, please make sure the content conforms to the `Lesson development` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [x] If you are making a new episode, please make sure the content conforms to the `Lesson organization` and `Lesson formatting` sections in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [x] Preferably Codacy checks pass. Status can be seen below your pull request. If there is an error, click the link to find out why.
- [x] Preview changes on your machine before pushing them to GitHub by running `make serve`, alternatively `make docker-serve`. Please see the `Previewing your changes locally` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) for installation instructions.
- [x] All code instructions have been tested.

If you need help with any of the tasks above, please do not hesitate to ask by commenting in the issue or pull request.

* * *

Closes #245

### CODE REVIEW

File: _episodes/00-quickstart.md
Comment (SarahAlidoost) [inline _episodes/00-quickstart.md:13]: I could not find where this objective is explained in this lesson.

File: _episodes/00-quickstart.md
Comment (SarahAlidoost) [inline _episodes/00-quickstart.md:18]: Why we point the user to the installation guide? The main idea in issue #245 is that esmvaltool is already installed.

File: _episodes/00-quickstart.md
Comment (SarahAlidoost) [inline _episodes/00-quickstart.md:19]:
```suggestion
- "The information needed by ESMValTool has to be set up using `config-user.yml file`, uncommenting the Site-specific entries."
```

Comment (SarahAlidoost) [reply _episodes/00-quickstart.md:19]: Please use `environment` only for `conda environment`. You can use `site` or `infrastructure` for the machine where esmvaltool is installed.

File: _episodes/00-quickstart.md
Comment (SarahAlidoost) [inline _episodes/00-quickstart.md:27]: The main idea in issue https://github.com/ESMValGroup/ESMValTool_Tutorial/issues/245 is that esmvaltool is already installed. Can we replace this with a note something like "we assume that you are working on a compute clusters where esmvaltool is installed and you have access to data. with a link to Setup https://esmvalgroup.github.io/ESMValTool_Tutorial/setup.html

File: _episodes/00-quickstart.md
Comment (SarahAlidoost) [inline _episodes/00-quickstart.md:22]: Plaese add an introductory paragraph about what the learner will learn in this lesson, and who are the target audience.

File: _episodes/00-quickstart.md
Comment (SarahAlidoost) [inline _episodes/00-quickstart.md:35]: Please link to the `configuration` episode instead, see https://esmvalgroup.github.io/ESMValTool_Tutorial/03-configuration/index.html

File: _episodes/00-quickstart.md
Comment (SarahAlidoost) [inline _episodes/00-quickstart.md:76]: Please explain why users should set these things.

File: _episodes/00-quickstart.md
Comment (SarahAlidoost) [inline _episodes/00-quickstart.md:107]:
```suggestion
> ## Outputs of a Recipe
```

File: _episodes/00-quickstart.md
Comment (SarahAlidoost) [inline _episodes/00-quickstart.md:109]:
```suggestion
> The outputs of a recipe can be found in the `output_dir` specified in the 
```

File: _episodes/00-quickstart.md
Comment (SarahAlidoost) [inline _episodes/00-quickstart.md:2]:
```suggestion
title: "Quickstart guide"
```

File: _episodes/00-quickstart.md
Comment (SarahAlidoost) [inline _episodes/00-quickstart.md:4]: Will you add `exercises` to this lesson?

**Review by SarahAlidoost — CHANGES_REQUESTED** [review]
@mo-tgeddes thanks for developing the lesson :+1: . Please see my comments. Also, as we discussed in #245, it would be better if users first get an introduction to esmvaltool. So, let's, move this episode after the Introduction episode.

File: _episodes/00-quickstart.md
Comment (rswamina) [inline _episodes/00-quickstart.md:8]: How do I configure ESMValTool?

**Comment by rswamina** [comment]
It seems to me that a large part of the content here is already shared in other tutorial episodes (`Installation`, `Configuration` etc.) and readily seen too. I raised this point earlier but I am still not completely convinced about having all this content in a Quick Start episode when it is quite reaily accessible already. 
@mo-tgeddes  - can you help me understand why this separate lesson is necessary ? Is it to gather bits from existing episodes for a quicker start?  I am not quite sure if that works.

**Comment by ehogan** [comment]
Hi @SarahAlidoost @rswamina, this PR is still a work in progress. We realise we made some assumptions that do not align with the expected development workflow of this repository. I, personally, would not review a PR until I was requested to do so (e.g. via the PR in the top right corner), and so we had made the assumption that this PR wouldn't be reviewed until we requested a review. We have now changed this PR to a draft PR, to make it clear that it is still a work in progress (apologies for not doing this in the first place). May I suggest, in an attempt to prevent this from happening again (and to avoid people from wasting time reviewing PRs that aren't ready to be reviewed), that we creating some "Development Workflow" guidelines to describe the preferred process? I would be happy to start documenting some thoughts on a wiki page, if that's helpful? :)

**Comment by SarahAlidoost** [comment]
> Hi @SarahAlidoost @rswamina, this PR is still a work in progress. We realise we made some assumptions that do not align with the expected development workflow of this repository. I, personally, would not review a PR until I was requested to do so (e.g. via the PR in the top right corner), and so we had made the assumption that this PR wouldn't be reviewed until we requested a review. We have now changed this PR to a draft PR, to make it clear that it is still a work in progress (apologies for not doing this in the first place). May I suggest, in an attempt to prevent this from happening again (and to avoid people from wasting time reviewing PRs that aren't ready to be reviewed), that we creating some "Development Workflow" guidelines to describe the preferred process? I would be happy to start documenting some thoughts on a wiki page, if that's helpful? :)

no problem. Looking forward to improvements. Regarding your suggestion on "Development Workflow", there are already some guides in `CONTRIBUTING.md`, see [how-to-contribute](https://github.com/ESMValGroup/ESMValTool_Tutorial/blob/main/CONTRIBUTING.md#how-to-contribute). However, now that the pull request is marked as `draft`, we now it is a work in progress.

**Comment by ehogan** [comment]
> > Hi @SarahAlidoost @rswamina, this PR is still a work in progress. We realise we made some assumptions that do not align with the expected development workflow of this repository. I, personally, would not review a PR until I was requested to do so (e.g. via the PR in the top right corner), and so we had made the assumption that this PR wouldn't be reviewed until we requested a review. We have now changed this PR to a draft PR, to make it clear that it is still a work in progress (apologies for not doing this in the first place). May I suggest, in an attempt to prevent this from happening again (and to avoid people from wasting time reviewing PRs that aren't ready to be reviewed), that we creating some "Development Workflow" guidelines to describe the preferred process? I would be happy to start documenting some thoughts on a wiki page, if that's helpful? :)
> 
> no problem. Looking forward to improvements. Regarding your suggestion on "Development Workflow", there are already some guides in `CONTRIBUTING.md`, see [how-to-contribute](https://github.com/ESMValGroup/ESMValTool_Tutorial/blob/main/CONTRIBUTING.md#how-to-contribute). However, now that the pull request is marked as `draft`, we now it is a work in progress.

Apologies for totally missing the `CONTRIBUTING.md` file! I'd be happy to add a sentence or two about draft PRs and reviewing (via a new issue), if you'd like? :)

File: _episodes/00-quickstart.md
Comment (ehogan) [inline _episodes/00-quickstart.md:40]:
```suggestion
>   already been installed at the site where ESMValTool will be run. If this is
>   not the case, see the [Installation][lesson-installation] episode in this
>   tutorial.
```

File: _episodes/00-quickstart.md
Comment (ehogan) [inline _episodes/00-quickstart.md:42]:
```suggestion
> - Activate the ESMValTool environment by following the instructions at
```

File: _episodes/00-quickstart.md
Comment (ehogan) [inline _episodes/00-quickstart.md:28]: I wonder whether we should add a note to say what version of ESMValTool this quick start guide is compatible with?
```suggestion
---
> ## Compatibility
>
> This quickstart guide is compatible with ESMValTool v2.6.0.
 {: .callout}
```

**Review by ehogan — CHANGES_REQUESTED** [review]
Thanks @mo-tgeddes! :)

File: _episodes/00-quickstart.md
Comment (ehogan) [inline _episodes/00-quickstart.md:34]:
```suggestion

> ## What is the purpose of the quickstart guide?
```

File: _episodes/00-quickstart.md
Comment (ehogan) [inline _episodes/00-quickstart.md:38]:
```suggestion
> - The purpose of the quickstart guide is to enable a user of ESMValTool to
>   run ESMValTool as quickly as possible by making the bare minimum number of 
>   changes.
```

**Review by ehogan — APPROVED** [review]
Just two more very minor suggestions :)

**Comment by SarahAlidoost** [comment]
> Apologies for totally missing the `CONTRIBUTING.md` file! I'd be happy to add a sentence or two about draft PRs and reviewing (via a new issue), if you'd like? :)

no need to apologize, if you think contributors (and the esmvaltool community) will benefit from it, then why not? please go ahead with fixing the contributing guide.

**Review by ehogan — APPROVED** [review]
Great job @mo-tgeddes, thank you! 😁

**Review by mo-gregmunday — APPROVED** [review]
This all looks fantastic, and even more succinct than the original. Thanks Theo!!

**Comment by mo-tgeddes** [comment]
Hi, sorry for the earlier confusion. Two people at the Met Office have approved it so I have now marked it as ready for review. Thanks. :)

File: _episodes/01-quickstart.md
Comment (remi-kazeroni) [inline _episodes/01-quickstart.md:8]:
```suggestion
- "How do I load ESMValTool and check the installation?"
```

File: _episodes/01-quickstart.md
Comment (remi-kazeroni) [inline _episodes/01-quickstart.md:49]:
```suggestion
> - Load ESMValTool by following the instructions at
```

**Review by remi-kazeroni — COMMENTED** [review]
Thanks a lot for your contribution @mo-tgeddes! I have a few minor comments, feel free to take them into account. I think this episode can be useful for users who have access to an existing installation of ESMValTool. I like the fact that it does not refer to any machine/site in particular but could be used in all places with some kind of a central installation.

I would just recommend not to refer to the "ESMValTool environment" throughout the episode because this sounds like the user has to activate some kind of conda environment. This is not really the case for the main use case considered in this episode, e.g. loading a module. We could also think of other non-conda related cases where users would have access to containers/images, etc.

File: _episodes/01-quickstart.md
Comment (mo-tgeddes) [reply _episodes/01-quickstart.md:78]: The command is already in the recipe episode and I was trying not to repeat too much.

File: _episodes/01-quickstart.md
Comment (mo-tgeddes) [reply _episodes/01-quickstart.md:25]: The modules documentation says the purpose of the module load command is to update the environment. See https://modules.readthedocs.io/en/latest/module.html. Since the documentation says environment, I think we should also use it to avoid confusion. If activate sounds too conda specific I could change it to "load the ESMValTool environment" as a compromise?

File: _episodes/01-quickstart.md
Comment (mo-tgeddes) [reply _episodes/01-quickstart.md:56]: When I run `esmvaltool --help` this prints out: INFO: Showing help with the command 'esmvaltool -- --help'.
So I assumed that was the correct command as when I run it that info line goes away.

File: _episodes/01-quickstart.md
Comment (mo-tgeddes) [reply _episodes/01-quickstart.md]: d252a44 - done

File: _episodes/01-quickstart.md
Comment (remi-kazeroni) [reply _episodes/01-quickstart.md:34]: I think it would be simpler not to have the compatibility note for this episode because the instructions work with any v2+ version and that would be a way to keep the quickstart guide general. Also, it would not be necessary to update this episode after each ESMValTool release. The situation is different for other episodes where instructions and log messages are version dependent.

File: _episodes/01-quickstart.md
Comment (remi-kazeroni) [reply _episodes/01-quickstart.md:56]: Indeed, this is the way help messages are expected by the Fire module, see https://google.github.io/python-fire/guide/. But since `esmvaltool --help` is used throughout the tutorial (e.g. [here](https://esmvalgroup.github.io/ESMValTool_Tutorial/02-installation/index.html)) and our docs (e.g. [here](https://docs.esmvaltool.org/en/latest/quickstart/installation.html)), I think it would be more consistent and user-friendly to only use that command.

**Review by remi-kazeroni — COMMENTED** [review]
Thanks for clarifying @mo-tgeddes! This looks good to me. I have tried to clarify a few comments that I wrote previously. Please let me know if further input from me are needed. I agree with your suggestion to use "load the ESMValTool environment". This doesn't sound too conda specific but would also work for users having access to some central esmvaltool conda environment.

File: _episodes/01-quickstart.md
Comment (mo-tgeddes) [reply _episodes/01-quickstart.md]: 9f31e4f changed it.

File: _episodes/01-quickstart.md
Comment (mo-tgeddes) [reply _episodes/01-quickstart.md]: changed to load and check environment - 9f31e4f.

File: _episodes/01-quickstart.md
Comment (mo-tgeddes) [reply _episodes/01-quickstart.md]: Load environment 9f31e4f.

File: _episodes/01-quickstart.md
Comment (mo-tgeddes) [reply _episodes/01-quickstart.md]: 430b245 done.

File: _episodes/01-quickstart.md
Comment (mo-tgeddes) [reply _episodes/01-quickstart.md:30]: Done 8ac17f7.

File: _episodes/01-quickstart.md
Comment (remi-kazeroni) [inline _episodes/01-quickstart.md:38]: Perhaps
```suggestion
> ## How do I load and check the ESMValTool environment?
```
to repeat what's in the objectives?

File: _episodes/01-quickstart.md
Comment (remi-kazeroni) [inline _episodes/01-quickstart.md:24]: Does the "installation guide" here refer to the installation episode or the Read the Docs page on ESMValTool installation? Perhaps you could add a hyperlink to the guide.

**Review by remi-kazeroni — APPROVED** [review]
Thanks for all your work and for addressing my comments @mo-tgeddes. I have 2 very minor points just for the sake of completeness but this episode looks fine to me as it is 👍

File: _episodes/01-quickstart.md
Comment (mo-tgeddes) [reply _episodes/01-quickstart.md]: Changed it to installation episode and added link 1c3cfb5.

**Review by LisaBock — CHANGES_REQUESTED** [review]
Thanks @mo-tgeddes ! This episode is clear and short. 
But I wonder if their is the idea that the quickstart guide should be done by everyone? Or if it can be skipped if someone plans to do the entire tutorial. I am not totally sure but maybe we could mention something linke that in the "What is the purpose of the quickstart guide?" section?

File: _episodes/01-quickstart.md
Comment (mo-tgeddes) [reply _episodes/01-quickstart.md:25]: 9d67a5c done.

File: _episodes/04-recipe.md
Comment (mo-tgeddes) [reply _episodes/04-recipe.md:26]: Thanks for pointing that out. I have fixed it now 4b76f78.

**Comment by mo-tgeddes** [comment]
> Thanks @mo-tgeddes ! This episode is clear and short. But I wonder if their is the idea that the quickstart guide should be done by everyone? Or if it can be skipped if someone plans to do the entire tutorial. I am not totally sure but maybe we could mention something linke that in the "What is the purpose of the quickstart guide?" section?

I've added a line to say that you don't have to do the tutorial if you are doing the quickstart guide. Hopefully that makes it clearer :) cc47081.

**Review by LisaBock — APPROVED** [review]

**Comment by LisaBock** [comment]
Hi @SarahAlidoost ! As you requested changes in the beginning for this pull request we have to wait for your approval.

File: _episodes/00-quickstart.md
Comment (SarahAlidoost) [reply _episodes/00-quickstart.md]: One suggestion for lesson development in the future: if learners want to practice the skills with the exercises, there should be a solution/answer. For example, when we asked them to run `esmvaltool --help`, we should also tell them what would be the output. More examples can be found in other episodes such as [03-configuration](https://esmvalgroup.github.io/ESMValTool_Tutorial/03-configuration/index.html). Also for more information see [Designing challenges](https://carpentries.github.io/curriculum-development/designing-challenges.html#designing-challenges-1).

**Review by SarahAlidoost — APPROVED** [review]
@mo-tgeddes nice work, and thank you for fixing the lesson. :+1:

**Pull request merged**
